demo/angularjs/components/expansion-panel/doc.md

```diff
@@ -0,0 +1,23 @@
+# Expansion panel
+
+**Expansion panels deliver large amounts of content in a small space through <a href="https://en.wikipedia.org/wiki/Progressive_disclosure" target="_blank">progressive disclosure</a>.**
+
+## Default
+
+<demo-block component="expansion-panel" partial="default"></demo-block>
+
+## Drag option
+
+A drag handle can be added.
```

Comment:
Explain how the drag handle behaves. Is it always horizontal or vertical or both? Why should developers use it?